python-libraries/nanover-omni/src/nanover/omni/openmm.py

```diff
@@ -18,6 +18,9 @@
 class OpenMMSimulation:
     """
```

Comment:
"""
    A wrapper for OpenMM simulations to run inside the OmniRunner.

    The following attributes can be configured after construction:
    - :attr:`frame_interval`: Number of simulation steps to advance between frames.
    - :attr:`include_velocities`: Include particle velocities in frames.
    - :attr:`include_forces`: Include particle forces in frames.
    - :attr:`platform_name`: Name of OpenMM platform to use when loading the system from XML.
    """